tensorflow/tools/dockerfiles/tests/import-onednn.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+# Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ============================================================================
+
+{ # try
```

Comment:
This code is pretty different from the other tests (I have never seen bash try-catch emulated like this before). Can it be rewritten to look more like a typical bash script?

E.g. 
```bash
python -c 'from tensorflow.python ...'
new_mkl_enabled=$?
python -c 'from tensorflow.python ...'
old_mkl_enabled=$?
if [[ $new_mkl_enabled -eq 0 ]]; then
   echo "Pass ..."
elif [[ $old_mkl_enabled -eq 0]]; then
   echo "Pass..."
else
   die "Fail..."
fi
```

Reply:
Sure I can just do it the old fashioned way. I thought this way we cover all scenarios for old and new TF since the API's are different between `v1` and `v2`. 

Do you still want me to change it?

Reply:
It's possible to have the same check functionality, just with a different bash style, right? My sample was just meant to show an example of the structure I'm used to.

Reply:
Updated the test case so that it looks like the rest of them 🙂